src/main/java/io/antmedia/muxer/RecordMuxer.java

```diff
@@ -453,7 +453,7 @@ public synchronized void writeTrailer() {
 
 	}
 
-	public static void saveToStorage(String prefix, File fileToUpload, String fileName, StorageClient storageClient) {
+	public static void saveToStorage(String prefix, File fileToUpload, String fileName, StorageClient storageClient, String storageClass) {
```

Comment:
Giving storage class as a parameter to the Muxers is not a good idea. It should be just a parameter in the StorageClient